Use pyside python modules from build since they are not installed yet.
Reported upstream: https://bugreports.qt.io/browse/PYSIDE-1430 (pyside2)
Additional fix for pyside6

--- a/sources/pyside6/tests/init_paths.py.ORIG	2021-04-19 18:07:00.000000000 +0200
+++ b/sources/pyside6/tests/init_paths.py	2021-05-08 09:55:33.560511618 +0200
@@ -48,13 +48,12 @@
 
     python_dirs = [os.path.join(src_dir, 'util')]  # Helper module
 
-    pyside_build_dir = os.path.join(get_build_dir(), 'pyside6')
-    python_dirs.append(pyside_build_dir)   # for PySide6
-    lib_dirs = [os.path.join(pyside_build_dir, 'libpyside'),
-                os.path.join(pyside_build_dir, 'libpysideqml')]
+    python_dirs.append(get_build_dir())    # for PySide6
+    lib_dirs = [os.path.join(get_build_dir(), 'libpyside'),
+                os.path.join(get_build_dir(), 'libpysideqml')]
 
     if testbindings_module:
-        python_dirs.append(os.path.join(pyside_build_dir,
+        python_dirs.append(os.path.join(get_build_dir(),
                                         'tests', 'pysidetest'))
         lib_dirs.append(_get_qt_lib_dir())
 
